ui/src/app/pages/workspace/invalid-billing-banner.tsx

```diff
@@ -27,31 +28,33 @@ export const InvalidBillingBanner = fp.flow(
   const userAction = enableBillingUpgrade ?
     'Please provide a valid billing account or contact support to extend free credits.' :
     'Please contact support to extend free credits.';
-  return <StatusAlertBanner
```

Comment:
This is an **alert** but it's not a **status alert**, so it's not appropriate to call `StatusAlertBanner`.  What we want is a banner with the **same styling as** `StatusAlertBanner`.  That is now `ToastBanner` called with `warningStyle={true}`